pages/product-page/product-listing.css

```diff
@@ -0,0 +1,15 @@
+.divider {
+  border: 1px solid var(--regal-blue-dark);
+  width: 80%;
+  margin: auto;
+}
+.filters-sidebar-container {
+  height: 100%;
+  width: 20%;
+  /* border: 1px solid var(--regal-blue-dark); */
```

Comment:
Remove unused code.

Reply:
okay will make these changes , thanks for reviewing the code 😄